src/lib/transaction_snark_scan_state/transaction_snark_scan_state.ml

```diff
@@ -282,7 +283,7 @@ end = struct
           val verify :
                Ledger_proof.t
             -> Ledger_proof_statement.t
-            -> message:Sok_message.t
+            -> message:Sok_message.Stable.Latest.t
```

Comment:
`Stable.Latest` shouldn't be used in function calls just in case it updates. Does `~message` need to be on the stable one? Would `Sok_message.t` work here?

Reply:
Yes, there's a type equation that should make it work

My error, I put that in before add the equation